README.md

```diff
@@ -23,5 +23,23 @@ If two dates are given then all the bugs found on those days and between (fully
 ./ubuntu_server_triage.py 2016-09-10 2016-09-12
 ```
 
+## Open Bugs in Browser
+Quite commonly the triager wants to open all bugs in the browser, to read, review and manage them. Via ``open`` argument that can be done automatically.
+```
+./ubuntu_server_triage.py --open 2016-09-10 2016-09-12
+```
+
+## Launchpad Name and subscription type
+By default this searches for the structural subscription of the ubuntu-server Team.
+But depending on the use case one might overwrite the team name with `--lpname` (which can be any launchpad user, doesn't have to be a Team).
+Additionally, especially when setting a personal name it is common that the filter should be switched to check for bug subscription instead of a structural subscription which can be done via `--bugsubscriber`.
```

Comment:
See my comment below, but I am confused by structural subscription. This sounds like the ones where the user has specifically added their name, as opposed to subscribed due to a list like ubuntu-server-bugs?